### PR TITLE
[chore] Add @bryan-aguilar as a triager

### LINKED ISSUE #24875: Request to add @bryan-aguilar as triager
### Component(s)

_No response_

### Describe the issue you're reporting

I would like to volunteer as a triager. I have been active in the opentelemetry community for a while and would like to contribute more to the collector repositories.

Contributions in collector-contrib
[Pull requests](https://github.com/open-telemetry/opentelemetry-collector-contrib/pulls?q=is%3Apr+author%3Abryan-aguilar)
[Issues](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aissue+author%3Abryan-aguilar)

### PR DESCRIPTION
**Pull request by bryan-aguilar**
**Description:** I volunteer as ~~tribute~~ triager. 

**Link to tracking Issue:** Fixes #24875

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]
👍

**Review by songy23 — APPROVED** [review]
Welcome!

**Review by atoulme — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Triagers ([@open-telemetry/collector-contrib-triagers](https://github.com/orgs/o
 - [Jared Tan](https://github.com/JaredTan95), DaoCloud
 - [Matt Wear](https://github.com/mwear), Lightstep
 - [Murphy Chen](https://github.com/Frapschen), DaoCloud
+- [Bryan Aguilar](https://github.com/bryan-aguilar), AWS
```
Comment (evan-bradley) [inline README.md:83]: Normally these go in alphabetical order by first name.

**Review by evan-bradley — APPROVED** [review]

**Comment by dmitryax** [comment]
Welcome @bryan-aguilar :)

**Pull request merged**